lib/board.rb

```diff
@@ -0,0 +1,37 @@
+class Board
+  attr_reader :cells
+
+  WINNING_COMBOS = [
+    [0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6],
+    [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]
+  ].freeze
+  def initialize
+    @cells = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
+
+  def show
+    puts "#{cells[0]} | #{cells[1]} | #{cells[2]}"
+    puts '---+---+---'
+    puts "#{cells[3]} | #{cells[4]} | #{cells[5]}"
+    puts '---+---+---'
+    puts "#{cells[6]} | #{cells[7]} | #{cells[8]}"
+  end
```

Comment:
No `puts/print` and `gets.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Kindly move it to `main.rb`